ApplicationLibCode/ProjectDataModel/Summary/RimSummaryRegressionAnalysisCurve.cpp

```diff
@@ -110,11 +117,54 @@ RimSummaryRegressionAnalysisCurve::~RimSummaryRegressionAnalysisCurve()
 //--------------------------------------------------------------------------------------------------
 void RimSummaryRegressionAnalysisCurve::onLoadDataAndUpdate( bool updateParentPlot )
 {
+    auto xValues    = RimSummaryCurve::valuesX();
+    auto yValues    = RimSummaryCurve::valuesY();
+    auto timeStepsX = RimSummaryCurve::timeStepsX();
+    auto timeStepsY = RimSummaryCurve::timeStepsY();
+
+    if ( axisTypeX() == RiaDefines::HorizontalAxisType::SUMMARY_VECTOR )
+    {
+        // NB! Assume that time stamps for X and Y are the same
+        std::vector<size_t> indicesToRemove;
```

Comment:
I think this assumption will not always be correct since you can have different cases on the different axis?
Maybe not important, but maybe try to detect when the assumption is wrong.

Reply:
Fixed, resampling will require some logic. Log error and do nothing if this situation is detected.